src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -118,3 +118,25 @@ void GraphingCalculator::TextBoxGotFocus(TextBox^ sender, RoutedEventArgs^ e)
 {
     sender->SelectAll();
 }
+
+void GraphingCalculator::ZoomInButtonClick(Object ^ /* parameter */)
+{
+    constexpr double scrollDamper = 0.15;
+    double scale = 1.0 + (abs(zoomButtonDelta) / WHEEL_DELTA) * scrollDamper;
```

Comment:
Since in this case, zoomButtonDelta never changes, you should just do this calculation once and hard code it here. Or make this a constexpr so that the value is computed at compile time.